Pod/Classes/SKYContainer.h

```diff
@@ -183,6 +183,22 @@ typedef void (^SKYContainerUserOperationActionCompletion)(SKYUser *user, NSError
       completionHandler:(SKYContainerUserOperationActionCompletion)completionHandler;
 
 /**
+ Creates a user account with the specified username, password and profile.
+ */
+- (void)signupWithUsername:(NSString *)username
+                  password:(NSString *)password
+                   profile:(NSDictionary *)profile
```

Comment:
I prefer to call this `profileDictionary` instead of `profile`. The reason is that we are likely to have a class `SKYProfile` (ref https://github.com/SkygearIO/features/issues/48) and I want the API to be clear that this method accepts a dictionary instead of a profile class.